addons/sale_timesheet/migrations/11.0.1.0/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,66 @@
+---Fields in module 'sale_timesheet'---
+sale_timesheet / account.analytic.line    / timesheet_invoice_id (many2one): NEW relation: account.invoice
+# Nothing to do
```

Comment:
Both this field and `timesheet_revenue` (and getting also o2m field `timesheet_ids`) should be computed for old entries. This can be done calling:
```python
env['account.invoice'].search([
    ('invoice_line_ids.product_id.type', '=', 'service'),
])._compute_timesheet_revenue()
```